Sources/LeafKit/LeafSyntax/LeafSyntax.swift

```diff
@@ -135,24 +344,73 @@ extension Syntax {
         }
     }
 
-    public final class Conditional {
-        public let condition: ConditionalSyntax
-        public internal(set) var body: [Syntax]
-        public private(set) var next: Conditional?
+    public struct Conditional: BodiedSyntax {
+        public internal(set) var chain: [(
+            condition: ConditionalSyntax,
+            body: [Syntax]
+        )]
```

Comment:
Syntax.Conditional is now a struct containing a chain of (ConditionalSyntax, Body) block instead of a set of linked class instances.